bootcd/rpms/genesis_scripts/src/genesis.init

```diff
@@ -0,0 +1,25 @@
+#! /bin/bash
+#
+# chkconfig: 2345 09 89
+# description: run genesis bootloader
+#
+
+### BEGIN INIT INFO
+# Required-Start:    $local_fs $network $syslog sshd udev-post
+# Default-Start:     2 3 4 5
+# Description:       run genesis bootloader
+### END INIT INFO
+
+. /etc/rc.d/init.d/functions
+
+case "$1" in
+    start)
+	echo -n $"Starting genesis bootloader:  "
+	/usr/bin/run-genesis-bootloader &>/dev/null &
```

Comment:
Simplest solution seems to me to just use `nohup`.
Adding it doesn't seem to break anything, but I haven't done extensive testing on running the init script from the command line.
I can't  think of a good reason someone would want to do that.
Either it ran correctly from `init` at boot time or you're doing manual debugging running `genesis` manually...

I've pushed an updated commit that uses nohup here.